tagger/plot/common.py

```diff
@@ -0,0 +1,62 @@
+#Plotting
+import numpy as np
+import matplotlib.pyplot as plt
+import matplotlib
+import mplhep as hep
+plt.style.use(hep.style.ROOT)
+import matplotlib.pylab as pylab
+params = {'legend.fontsize': 'medium',
+         'axes.labelsize': 'x-large',
+         'axes.titlesize':'x-large',
+         'xtick.labelsize':'medium',
+         'ytick.labelsize':'medium'}
+pylab.rcParams.update(params)
+
+#GLOBAL VARIABLES TO USE ACROSS PLOTTING TOOLS
+MINBIAS_RATE = 32e+3 #32 kHZ
+
+# Define pT bins
+PT_BINS = np.array([
+    15, 17, 19, 22, 25, 30, 35, 40, 45, 50,
+    60, 76, 97, 122, 154, 195, 246, 311,
+    393, 496, 627, 792, 1000
+])
+
+WPs_CMSSW = {
+    'tau': 0.22,
```

Comment:
For these add an attachment, as per the seeded cone reco pt cuts below or some description of where they come from (e.g. link to CMSSW file) so we can refer back to these values. In the future we should define these in their own config file so that we don't need to change the python code to change these constants

Reply:
Yea, I'm not really sure I figured out the best solution for this, I'll add the comment. But yes in the future we should have a better solution.